Tasks/GooglePlayRelease/googleutil.ts

```diff
@@ -113,7 +112,7 @@ export interface GlobalParams {
     params?: PackageParams;
 }
 
-export function getJWT(key: ClientKey): JWT {
+export function getJWT(key: ClientKey): any {
```

Comment:
I know, it's a really wrong idea, but it does work without any sort of problems ¯\_(ツ)_/¯
I've tested and I perfectly got the inAppUpdatePriority in my app with PlayCore

Reply:
I've taken a look at the source code here: `googleapis-common` uses `google-auth-library` ^5.6.1 under the hood to provide the `JWT` class:

![image](https://user-images.githubusercontent.com/72494759/113416651-71642f00-93ca-11eb-9724-b463947bb54a.png)

![image](https://user-images.githubusercontent.com/72494759/113416133-53e29580-93c9-11eb-9769-8e4c777a299c.png)

Since we're using `google-auth-library` ^2.0.0 in our task, we have some incompatibilities. I think in this case it makes sense to bump `google-auth-library` in our task to ^5.6.1, see suggestion below